core/src/main/scala/org/apache/spark/ui/jobs/ExecutorTable.scala

```diff
@@ -70,6 +70,7 @@ private[ui] class ExecutorTable(stage: StageData, store: AppStatusStore) {
           Blacklisted
           </span>
         </th>
+        <th>Logs</th>
```

Comment:
What about putting the logs link after the ID column, to stay more consistent? or is it that the Logs column is last in other tables?

Reply:
In other tables, like Executors tables, Logs column is in the last. That is why I followed the same here.
![screenshot from 2018-11-14 23-29-43](https://user-images.githubusercontent.com/23054875/48502307-3565b900-e865-11e8-84fa-6854607526a4.png)